scriptmodules/supplementary/runcommand.sh

```diff
@@ -19,7 +19,7 @@ function _update_hook_runcommand() {
 }
 
 function depends_runcommand() {
-    local depends=()
+    local depends=(python-pyudev)
```

Comment:
as we use joy2key for retropie-setup navigation also, the dependency may be better to put in the main retropie-setup script. I was also thinking we could do it without the module perhaps - noticed http://stackoverflow.com/questions/43599936/pyudev-how-to-get-the-devices-friendly-name while I was googling ;-)